packages/editor/src/index.tsx

```diff
@@ -18,31 +18,34 @@ import { invokeGlobalErrorHandler } from 'common/lib/utilities/splash.screen';
 
 import Pages from './pages';
 
-if (!isRedirectingAway) {
-  document.addEventListener(
-    'keydown',
-    e => {
-      if (
-        e.keyCode === 83 /*s key*/ &&
-        (navigator.platform.match('Mac') ? e.metaKey : e.ctrlKey)
-      ) {
-        e.preventDefault();
+(async () => {
```

Comment:
The changes in this file and the next one are merely indentation (caused by me needing to do `(async () { ... })();` to allow me to use "await" inside.

Reply:
I personally prefer the use of .then instead of a self executing anonymous async function in this instance